src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -32,12 +37,14 @@ import {
   textStyle
 } from './ChatItemText.style';
 import {replyIcon} from './ChatItemTargetText';
+import {formatBytes} from '../../../utils/string/StringUtils';
 
 const ChatItemMyTextV2 = ({
   username = 'Anonymous User',
   time = '4h',
   isContinuous = false,
   message = '',
+  attachments = [],
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The default value for `attachments` is an empty array. This is a good practice as it ensures that `attachments` will always be iterable, preventing potential runtime errors when trying to access methods like `.length` or `.map()` on `undefined`.



<!-- This is an auto-generated comment by OSS CodeRabbit -->